### DISCUSSION TITLE
Add a button to Hide Search Matches highlights

### DISCUSSION THREAD

**Discussion by jeremy-feng** [squidfunk/mkdocs-material#5827] [Ideas]
I hope that there will be a button that can hide search matches highlights. I saw this feature from [Jupyter book](https://jupyterbook.org/en/stable/intro.html).

![image](https://github.com/squidfunk/mkdocs-material/assets/44312563/dcf13a4a-4f6e-4aa2-b51d-a593f9dc4b6f)

**Comment by squidfunk** [comment]
Jup, this is coming at some point when we tackle #4125, there's already a prototype for [better search result navigation](https://github.com/squidfunk/mkdocs-material/issues/4125#issuecomment-1374882924).

**Reply by jeremy-feng** [reply]
Thank you! Looking forward to trying it out.
